Package.swift

```diff
@@ -175,7 +182,15 @@ let authTargets: [Target] = [
             "AWSCognitoAuthPlugin",
             "AWSPluginsTestCommon"
         ],
-        path: "AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests"
+        path: "AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests",
```

Comment:
These changes are temporary added to aid in testing and generating small PRs. Will be removed before merging to main dev-preview branch